sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineOptions.java

```diff
@@ -67,7 +56,7 @@
 
   void setClock(Clock clock);
 
-  @Default.Boolean(false)
+  @Default.Boolean(true)
```

Comment:
Seems like a semantic change for users who weren't setting this -- why is this OK?